bin/sidecli.re

```diff
@@ -199,6 +206,12 @@ let create_transaction =
   Lwt.return(`Ok());
 };
 
+let endpoint_uri = {
```

Comment:
should be changed to `transaction_endpoint_uri` or `doc` should be changed not to specify that it's used for a transaction (possibly make `endpoint_uri` take a parameter?)

Reply:
I have incorporated Eduardo's suggestion ie using identity.json